From 2f0d42d6999f1b05d801efaac757a51b96fe0c62 Mon Sep 17 00:00:00 2001
From: icg_intg <icg_intg@intel.com>
Date: Wed, 6 Jun 2018 10:39:06 +0800
Subject: [PATCH 1111/1240] media: intel-ipu4: css:
 scci_stable_iot_mr4_beta_20180506_20180605_1006

Message for Open Source:
New firmware css release.

Message for Internal:
[Feature]New firmware css release.
[Root Cause/Changes]
isys & psys package lib2600b0 for commit-id 127908d
FW_Manifest_ID: 4a17268b664d4ef0a45b37b03f20218bdabac0de
FW_Build_Branch: stable_iot_mr4_beta_20180506
FW_Build_Topic: auto_merge_queue
library version: commit-id 127908d
FW Bundle patch number of this release :  333826
FW tag : scci_stable_iot_mr4_beta_20180506_20180605_1006

Change-Id: Ibe844b5759a5c27041acb70af85d1646ecbf18fd
Depends-on: I71ab66e5d8908b6a94344b971d8e27db98d76ff2
Depends-on: I6beaff48c6e0008817839705976c0211fd98a767
Signed-off-by: icg_intg <icg_intg@intel.com>
Signed-off-by: Meng Wei <weix.meng@intel.com>
---
 drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h           | 2 +-
 .../lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h        | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h b/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
index eea56e7..081886c 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
+++ b/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
@@ -11,4 +11,4 @@
 * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
 * more details.
 */
-#define IA_CSS_FW_PKG_RELEASE  0x20180529
+#define IA_CSS_FW_PKG_RELEASE  0x20180605
diff --git a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
index eea56e7..081886c 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
+++ b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
@@ -11,4 +11,4 @@
 * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
 * more details.
 */
-#define IA_CSS_FW_PKG_RELEASE  0x20180529
+#define IA_CSS_FW_PKG_RELEASE  0x20180605
-- 
2.7.4

